### PR TITLE
translate teams page

### PR DESCRIPTION
**Pull request by wenzhenxiang**

### CODE REVIEW

File: zh-CN/teams.html [hunk 1]
```diff
@@ -120,7 +120,7 @@
                             </li>
                         </ul>
                     </div>
-                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">Claim KSM</a>
+                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">索取 KSM</a>
```
Comment (xlc) [inline zh-CN/teams.html]:
```suggestion
                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">领取 KSM</a>
```

File: zh-CN/teams.html [hunk 2]
```diff
@@ -223,21 +223,21 @@ <h5 class="mb-0">Polka.io</h5>
...
                             </a>
                         </div>
                         <div class="col-sm-12 col-md-6 col-lg-4">
                             <a href="https://www.mathwallet.org/" target="_blank" class="card card-sm card-body flex-row align-items-center hover-shadow-3d">
                                 <img src="assets/img/teams/mathwallet.png" alt="Math Wallet" class="icon icon-lg">
                                 <div class="ml-3">
-                                    <h5 class="mb-0">Math Wallet</h5>
-                                    <span>Wallet/KSM Claims Dapp</span>
+                                    <h5 class="mb-0">麦子钱包</h5>
+                                    <span>钱包/KSM 索取 Dapp</span>
```
Comment (xlc) [inline zh-CN/teams.html]:
```suggestion
                                    <span>钱包/KSM 领取 Dapp</span>
```

File: zh-CN/teams.html [hunk 3]
```diff
@@ -323,12 +323,12 @@ <h5 class="mb-0">Math Wallet</h5>
...
                                                     </g>
                                                 </svg>
-                                                <span>Subscribe</span>
+                                                <span>订阅</span>
                                             </button>
                                         </div>
                                     </div>
                                 </form>
-                                <small class="text-muted form-text">To see how we use your information please see our <a href="privacy.html" target="_blank">privacy policy</a>. Unsubscribe anytime by clicking the link in the bottom of any newsletter.
+                                <small class="text-muted form-text">想了解我们如何使用您的信息，请参阅我们的<a href="privacy.html" target="_blank">隐私政策</a>。随时取消订阅，点击任何通讯底部的链接。
```
Comment (xlc) [inline zh-CN/teams.html]:
```suggestion
                                <small class="text-muted form-text">想了解我们如何使用您的信息，请参阅我们的<a href="privacy.html" target="_blank">隐私政策</a>。可以在任何时候通过点击任意邮件通知底部的链接来取消订阅。
```

File: zh-CN/teams.html [hunk 4]
```diff
@@ -345,54 +345,54 @@ <h5 class="mb-0">Math Wallet</h5>
...
+                        Kusama 是一个实验性社区的研发网络。
                     </p>
                 </div>
                 <div class="col-6 col-sm-3 col-lg-2 mt-3 mt-lg-0 pl-2">
-                    <h5>Info</h5>
+                    <h5>信息</h5>
                     <ul class="nav flex-column">
                         <li class="nav-item">
-                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">Claim KSM</a>
+                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">索取 KSM</a>
```
Comment (xlc) [inline zh-CN/teams.html]:
```suggestion
                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">领取 KSM</a>
```

File: zh-CN/teams.html [hunk 5]
```diff
@@ -345,54 +345,54 @@ <h5 class="mb-0">Math Wallet</h5>
...
-                            <a href="https://guide.kusama.network/en/latest/" class="nav-link">User guide</a>
+                            <a href="https://guide.kusama.network/en/latest/" class="nav-link">用户指南</a>
                         </li>
                         <li class="nav-item">
-                            <a href="teams.html" class="nav-link">Teams</a>
+                            <a href="teams.html" class="nav-link">团队成员</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug bounty</a>
+                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug 奖励</a>
```
Comment (xlc) [inline zh-CN/teams.html]:
```suggestion
                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug 赏金计划</a>
```

File: zh-CN/teams.html [hunk 6]
```diff
@@ -345,54 +345,54 @@ <h5 class="mb-0">Math Wallet</h5>
...
                         <li class="nav-item">
-                            <a href="https://riot.im/app/#/room/#kusamawatercooler:polkadot.builders" target="_blank" class="nav-link">Chat</a>
+                            <a href="https://riot.im/app/#/room/#kusamawatercooler:polkadot.builders" target="_blank" class="nav-link">聊天室</a>
                         </li>
                         <li class="nav-item">
-                            <a href="contact.html" class="nav-link">Contact</a>
+                            <a href="contact.html" class="nav-link">联系方式</a>
                         </li>
                         <li class="nav-item">
                             <a href="logos.html" class="nav-link">Logos</a>
```
Comment (ansonla3) [inline zh-CN/teams.html]: 商标?

**Comment by lsaether** [comment]
Please resolve conflicts with master before the PR can be merged.

**Pull request merged**